composer.json

```diff
@@ -1,79 +1,85 @@
 {
     "name": "zendframework/zendframework",
-    "description": "Zend Framework 2",
-    "type": "library",
+    "description": "Zend Framework",
+    "type": "metapackage",
```

Comment:
Do we need to update .gitattributes to ensure that we don't install anything or is composer not that strict?